google-cloud-scheduler/src/test/java/com/google/cloud/scheduler/v1beta1/it/ITSystemTest.java

```diff
@@ -0,0 +1,155 @@
+ * limitations under the License.
+ */
+package com.google.cloud.scheduler.v1beta1.it;
+
+import static junit.framework.TestCase.assertEquals;
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.pubsub.v1.TopicAdminClient;
+import com.google.cloud.scheduler.v1beta1.CloudSchedulerClient;
+import com.google.cloud.scheduler.v1beta1.Job;
+import com.google.cloud.scheduler.v1beta1.JobName;
+import com.google.cloud.scheduler.v1beta1.LocationName;
+import com.google.cloud.scheduler.v1beta1.PubsubTarget;
+import com.google.cloud.scheduler.v1beta1.UpdateJobRequest;
+import com.google.common.collect.Lists;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.ProjectTopicName;
+import java.util.List;
+import java.util.UUID;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITSystemTest {
+
+  private static CloudSchedulerClient client;
+  private static TopicAdminClient publisherClient;
+
+  private static final String PROJECT = ServiceOptions.getDefaultProjectId();
+  private static final String LOCATION = "us-central1";
+  private static final String ID = UUID.randomUUID().toString().substring(0, 8);
+  private static final String JOB = "test-job-" + ID;
+  private static final String TOPIC = "test-topic-" + ID;
+  private static final String SCHEDULE = "* * * * *";
+  private static final String TIME_ZONE = "UTC";
+  private static final ProjectTopicName TOPIC_NAME = ProjectTopicName.of(PROJECT, TOPIC);
+  private static final String JOB_NAME = JobName.of(PROJECT, LOCATION, JOB).toString().trim();
+  private static final PubsubTarget PUB_SUB_TARGET =
+      PubsubTarget.newBuilder()
+          .setTopicName(TOPIC_NAME.toString())
+          .setData(ByteString.copyFrom("test".getBytes()))
+          .build();
+  private static final LocationName PARENT = LocationName.of(PROJECT, LOCATION);
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+
+    /** The job will be delivered by publishing a message to the given Pub/Sub topic. */
+    publisherClient = TopicAdminClient.create();
+    publisherClient.createTopic(TOPIC_NAME);
+
+    client = CloudSchedulerClient.create();
+    Job createJob =
+        Job.newBuilder()
+            .setName(JOB_NAME)
+            .setPubsubTarget(PUB_SUB_TARGET)
+            .setSchedule(SCHEDULE)
+            .setTimeZone(TIME_ZONE)
+            .build();
+    client.createJob(PARENT, createJob);
+  }
+
+  @AfterClass
+  public static void afterClass() {
+    client.deleteJob(JOB_NAME);
+    publisherClient.deleteTopic(TOPIC_NAME);
+    publisherClient.close();
+    client.close();
+  }
+
+  @Test
+  public void getJobTest() {
+    Job job = client.getJob(JOB_NAME);
+    assertEquals(JOB_NAME, job.getName());
+    assertEquals(Job.State.ENABLED, job.getState());
+    assertEquals(SCHEDULE, job.getSchedule());
+    assertEquals(TIME_ZONE, job.getTimeZone());
+    assertEquals(PUB_SUB_TARGET, job.getPubsubTarget());
+  }
```

Comment:
Does it make sense to introduce a meeting like:

```
private static checkJob(job) {
    assertEquals(JOB_NAME, job.getName());
    assertEquals(Job.State.ENABLED, job.getState());
    assertEquals(SCHEDULE, job.getSchedule());
    assertEquals(TIME_ZONE, job.getTimeZone());
    assertEquals(PUB_SUB_TARGET, job.getPubsubTarget());
}
```
and replace multiple checks with a single call?

Reply:
done